nss-day-cohort-45/tabloidfullstack-gifted-gingerbreads
# CRUD for Tags

# Add, Edit and Delete Tags

### How To Test
1. Fetch branch
2. See if you can see all the tags on the Tags page
3. Add a tag
4. Edit a tag
5. Delete a tag

Double check each time by checking the tags page
